File: src/gtfs/models/Manifest.py
from typing import Any

from pydantic import BaseModel


class Manifest(BaseModel):
    """Build manifest with metadata and checksums."""

    schema_version: int
    tool_version: str
    created_at_iso: str
    inputs: dict[str, Any]
    outputs: dict[str, str]  # filename -> sha256
    stats: dict[str, int]
    build: dict[str, str]

File: src/main/kotlin/com/raptor/gtfs/models/Manifest.kt
package com.raptor.gtfs.models

import kotlinx.serialization.Serializable

@Serializable
data class Manifest(
    val schema_version: Int,
    val tool_version: String,
    val created_at_iso: String,
    val inputs: Map<String, String>, // Changed from Any since JSON serializes map of strings mostly
    val outputs: Map<String, String>,
    val stats: Map<String, Int>,
    val build: Map<String, String>
)
